Sources/Benchmark/regexredux.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import Dispatch
+import _StringProcessing
+
+// The Computer Language Benchmarks Game
+// https://salsa.debian.org/benchmarksgame-team/benchmarksgame/
+//
+// contributed by Francois Green
+// variant concurrency added by Daniel Sell
+func nsregexRedux() {
+  let input = FileManager.default.contents(
+    atPath: "input500000.txt"
+  )!
+  
+  var sequence = String(data: input, encoding: .ascii)!
+  
+  let inputLength = input.count
+  
+  let regex: (String) -> NSRegularExpression = { pattern in
+    return try! NSRegularExpression(pattern: pattern, options: [])
+  }
+  
+  sequence = sequence.replacingOccurrences(of: ">[^\n]*\n|\n", with: "",
+                                           options: .regularExpression)
+  
+  let codeLength = sequence.utf8.count
+  
+  var resultLength = 0
+  
+  let group = DispatchGroup()
+  
+  group.enter()
+  DispatchQueue.global().async {
```

Comment:
If it's simpler, we can prototype or benchmark a single-threaded approach primarily.